testsuite/openmodelica/cppruntime/mslElectricalMachinesTest.mos

```diff
@@ -0,0 +1,30 @@
+// name: mslElectricalMachinesTest
+// keywords: init const string substring
+// status: correct
+// teardown_command: rm -f *AIMC_Transformer*
+// cflags: -d=newInst
+
+setCommandLineOptions("+simCodeTarget=Cpp");
```

Comment:
I would have used --simCodeTarget=Cpp as the + argument specifier has been deprecated for 10 or so years :)